src/mpi/coll/iallgatherv/iallgatherv_tsp_recexch.c

```diff
@@ -18,8 +18,8 @@ static int MPIR_TSP_Iallgatherv_sched_intra_recexch_data_exchange(int rank, int
 {
     int mpi_errno = MPI_SUCCESS;
     int mpi_errno_ret ATTRIBUTE((unused)) = MPI_SUCCESS;
-    int partner, offset, count;
-    int i, vtx_id;
+    int partner;
+    int count, offset, i, vtx_id;
```

Comment:
Did you mean to use `MPI_Aint`?

Reply:
count offset are calculated by MPII_Recexchalgo_get_count_and_offset, which returns the rank number and rank offset. I think int is fine.